content/how-to/cloud-servers/scheduled-images-faq/index.md

```diff
@@ -151,105 +141,95 @@ support.
 {{<accordion title=`What is the "retention" value?` col="in" href="accordion9">}}
 
 The retention value is the maximum number of scheduled images for that
-particular server that will be retained in your account.  Once the
+particular server that your account retains. After the
 number of scheduled images for this server reaches the retention value,
-the oldest scheduled image(s) will be deleted so that the total number
+the system deletes the oldest scheduled images so that the total number
 of scheduled images for this server does not exceed the retention value.
 
-The retention value is set on a per-server basis. Therefore, you may
+Because you set the retention value on a per-server basis, you can
 specify a different retention value for each server.
 {{</accordion>}}
 {{<accordion title="What is the maximum retention value?" col="in" href="accordion10">}}
 
-The maximum retention value in the Rackspace open cloud is 65535.
-(That's about 179 years of daily images.)
+The maximum retention value in the Rackspace Open Cloud is 65535.
+That's about 179 years of daily images.
 {{</accordion>}}
 {{<accordion title="When does automatic deletion occur?" col="in" href="accordion11">}}
 
 When a scheduled image of a server has successfully completed, the
 scheduled images service creates a list of all that server's scheduled
```

Comment:
change to:
"...creates a list of all the scheduled images for that server..."

Reply:
done